### DISCUSSION TITLE
Sonatype migrating to Central Portal from the OSSRH

### DISCUSSION THREAD

**Discussion by dordor12** [thriving-dev/java-library-template#36] [General]
Hi everyone,

I've recently embarked on the journey of publishing my first Java library. However, I encountered some hurdles with the Sonatype [registration ](https://central.sonatype.org/register/central-portal/)and [publishing ](https://central.sonatype.org/publish/publish-portal-gradle/)process. It appears that Sonatype has implemented changes to their entire registration and publishing mechanism. From what I've gathered, these changes seem to render much of the current documentation and templates obsolete or outdated.

Has anyone else faced similar challenges with these recent changes? I'm keen to understand how others are navigating this situation and whether there are any updated guides or tips to follow. Any shared experiences or insights would be greatly appreciated as I navigate through this process.

Thank you in advance for your help!

**Comment by almogtavor** [comment]
@dordor12 I didn't try to fully understand why it didn't work for me, but something indeed doesn't go smooth with the Sonatype uploading. @hartmut-co-uk I think addition documentation here would really help. Or maybe it's because of the latest changes Sonatype did (Jira depreciation etc)
